image/image.go

```diff
@@ -787,3 +788,65 @@ func copyLocalSnapFile(snapPath, targetDir string, info *snap.Info) (dstPath str
 	dst := filepath.Join(targetDir, filepath.Base(info.MountFile()))
 	return dst, osutil.CopyFile(snapPath, dst, 0)
 }
+
+func ValidateSeed(seedFile string) error {
+	seed, err := snap.ReadSeedYaml(seedFile)
+	if err != nil {
+		return err
+	}
+
```

Comment:
If you take my suggestion below you can initialize errors here:
```suggestion
var errs []error
```